todo_app/data/trello_items.py

```diff
@@ -0,0 +1,167 @@
+from datetime import datetime, date
+
+from flask import session
+import requests
+import json
+import os
+
+
+class List:
+    def __init__(self, id, name):
+        self.id = id
+        self.name = name
+
+    @classmethod
+    def from_trello_response(cls, response):
+        return cls(response['id'], response['name'])
+
+
+class Item:
+    def __init__(self, id, name, desc, id_list, due=None):
+        self.id = id
+        self.name = name
+        self.desc = desc
+        self.id_list = id_list
+        self.list = get_list(self.id_list)

```

Comment:
This sneaky little call off to Trello leads to my (admittedly quite bloated) Trello board taking about 10 seconds to load on your homepage because we're making one request to Trello for every card; like the [classic N+1](https://docs.sentry.io/product/issues/issue-details/performance-issues/n-one-queries/#:~:text=N%2B1%20queries%20are%20a,than%20doing%20it%20just%20once.) queries in a DB, this can quickly tank your performance!

It's not really important here, but worth noting that if we use the `/lists` endpoint from Trello with `cards=open` as a query parameter then it will return all the lists & cards in a single giant response, which we can iterate through (psuedo-code below) to handle this as a single request, which could massively help performance!

```python
lists = get_all_lists_with_cards
for list in lists:
    for card in list['cards']:
        item = Item.from_trello_response(card, list)
```